File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/dtos/menu/MenuGroupPortionDTO.java
package com.robinfood.core.dtos.menu;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;

@AllArgsConstructor
@Data
@JsonInclude(JsonInclude.Include.NON_NULL)
public class MenuGroupPortionDTO {

    private final Boolean isDefault;
    private final BigDecimal discount;
    private final Long id;
    private final String image;
    private final String name;
    private final Long parentId;
    private final BigDecimal premiumPrice;
    private final BigDecimal price;
    private final Integer position;
    private final String sku;
    private final Long unit;
    private final Double weight;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/dtos/menu/response/MenuGroupPortionDTO.kt
package com.robinfood.core.dtos.menu.response

import java.math.BigDecimal

data class MenuGroupPortionDTO(
    val default: Boolean,
    val discount: BigDecimal,
    val id: Long,
    val image: String,
    val name: String,
    val parentId: Long,
    val premiumPrice: BigDecimal,
    val price: BigDecimal,
    val position: Int,
    val sku: String,
    val unit: Long,
    val weight: Double
)